radio-service-api/src/main/java/com/pnu/dev/radioserviceapi/service/VideoApiServiceImpl.java

```diff
@@ -22,6 +23,7 @@
 
 import static java.util.Objects.nonNull;
 
+@Profile("local")
```

Comment:
Sorry, I mentioned the wrong class in the description.
I meant `VideoApiClient`.

But, anyway, we don't want the object of the real class to be loaded to the context, when we use this profile.
It have to be stub, for this profile and real object for default one.